draft-ietf-quic-transport.md

```diff
@@ -3422,10 +3422,7 @@ Frame {
 {: #frame-layout title="Generic Frame Layout"}
 
 The frame types defined in this specification are listed in {{frame-types}}.
-The Frame Type in ACK, STREAM, MAX_STREAMS, STREAMS_BLOCKED, and
-CONNECTION_CLOSE frames is used to carry other frame-specific flags. For all
-other frames, the Frame Type field simply identifies the frame.  These
-frames are explained in more detail in {{frame-formats}}.
+{{frame-types}} includes some summary information that is explained below.
```

Comment:
"{{frame-types}} lists and summarizes information about each frame type that is
defined in this specification. The summarized information in {{frame-types}} is
described after the table."